# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/gcc3.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Older gcc (<3.4) are not able to honor the .rodata .section specification
correctly:

../sysdeps/i386/i686/memcmp.S: Assembler messages: 
../sysdeps/i386/i686/memcmp.S:390: Error: can't resolve `.rodata'  

  Rene Rebe <rene@exactcode.de>

--- ./sysdeps/i386/i686/memcmp.S~	2004-04-02 01:28:37.000000000 +0200
+++ ./sysdeps/i386/i686/memcmp.S	2004-05-22 21:27:39.000000000 +0200
@@ -351,7 +351,6 @@
 	popl	%esi
 	RETURN
 
-	.section	.rodata
 	ALIGN (2)
 L(table_32bytes) :
 	.long	L(0bytes) - . + 0x0
